test/y2network/interface_config_builders/bonding_test.rb

```diff
@@ -109,6 +109,8 @@
 
     context "when the architecture is s390" do
       let(:s390) { true }
+      let(:interface1) { Y2Network::Interface.new("iface1", type: Y2Network::InterfaceType::QETH) }
+      let(:interface2) { Y2Network::Interface.new("iface1", type: Y2Network::InterfaceType::QETH) }
```

Comment:
np: "iface1" -> "iface2" ?

Reply:
It will not affect the test at all, so will fix only in master